examples/gcloud/grpc_mnist_server.py

```diff
@@ -49,6 +49,7 @@ def forward(self, x):
 def main():
 
     parser = argparse.ArgumentParser(description="Provide the configuration")
+    parser.add_argument("--total_iter", type=int, required=True)
```

Comment:
why don't we use the same name (`num_epochs`) as defined in `cfg`?

Reply:
done